src/core/ext/filters/server_config_selector/server_config_selector_filter.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <functional>
 #include <memory>
+#include <type_traits>
```

Comment:
why do we need this header?

Reply:
IWYU says we do (it's probably worth digging into at some point though)